schema/partner.js

```diff
@@ -26,7 +26,7 @@ import {
 const PartnerType = new GraphQLObjectType({
   name: 'Partner',
   interfaces: [NodeInterface],
-  isTypeOf: (obj) => has(obj, 'has_full_profile') && has(obj, 'shows_count'),
+  isTypeOf: (obj) => has(obj, 'has_full_profile') && has(obj, 'profile_banner_display'),
```

Comment:
This is to support `short` JSON. Since the way an object was fetched may reflect different sets of available properties, we should probably just use fields available in `short` JSON for all these kinds of methods just to cover us.